docs/config.json

```diff
@@ -509,6 +509,10 @@
               "slug": "/management/guides/joining-nodes-aws-ec2/",
               "forScopes": ["oss", "enterprise"]
             },
+            {
+              "title": "Joining Nodes via Azure",
```

Comment:
We now have three "Joining Nodes" guides (in addition to the original token-based "Adding Nodes" guide), so i think we should do this:
- Add another section to "Manage your Cluster" called "Joining Nodes"
- Move all of our "Joining Nodes..." guides into this section, including the "Adding Nodes" guide from "Manage your Cluster > Admin Guides". 
- Add redirects from the old locations of our "Joining Nodes"/"Adding Nodes" guides to the new locations
- Add an intro page to the "Joining Nodes" section that provides a quick overview of the different joining methods and indicates when you would use each one.

If you don't have time for this, let me know and I can create a GitHub issue for the reorganization and plan to work on it separately.

Reply:
I would appreciate creating the separate issue, if you don't mind.